omod-1.10/src/test/java/org/openmrs/module/attachments/rest/AttachmentRestController1_10Test.java

```diff
@@ -452,4 +449,16 @@ public void getAttachmentBytes_shouldDownloadFile() throws Exception {
 		
 	}
 	
+	@Test
+	public void doSearch() throws Exception {
+		Patient patient = Context.getPatientService().getPatient(2);
+		MockHttpServletRequest request = request(RequestMethod.GET, getURI());
+		request.addParameter("patient", patient.getUuid());
+		SimpleObject response = deserialize(handle(request));
+		LinkedHashMap<String, String> result = (LinkedHashMap<String, String>) ((ArrayList<LinkedHashMap>) response
+		        .get("results")).get(0);
+		Assert.assertEquals("application/octet-stream", result.get("bytesMimeType"));
+		Assert.assertEquals(ContentFamily.OTHER.toString(), result.get("bytesContentFamily"));
+	}
```

Comment:
@ivange94 this method is not properly named and is not adopting the given/when/then pattern like the other test methods in this class.

Reply:
@gitcliff can you take care of this? For me to do this will mean I'd have to go through the process of creating a PR to your branch again then pinging you to merge. You can just copy this https://github.com/gitcliff/openmrs-module-attachments/pull/1/commits/f8d860886e7dd238c467fef316b30239303435b1#diff-3673f95939fa7f7146dee8481d51e52cL395 and make sure it passes. 

For the method name, you could use something like `doSearch_shouldGetAttachmentsWithMimeTypesAndContentFamilySet`